app/models/mail_chimp_client.rb

```diff
@@ -5,63 +5,49 @@ class MailChimpClient
   ADOPTER_LIST_ID = '5e50e2be93'.freeze
 
   def initialize
-    @gibbon = Gibbon::API.new
+    @gibbon = Gibbon::Request.new
     @gibbon.timeout = 30
   end
 
+  def subscribe(list_id, email, merge_vars)
+    gibbon.lists(list_id).members(hashed(email)).upsert(
+      body: {
+        email_address: email,
+        status: 'subscribed',
+        merge_fields: merge_vars
+      }
+    )
+  end
+
+  def unsubscribe(list_id, email)
+    gibbon.throws_exceptions = false
+    gibbon.lists(list_id).members(hashed(email)).delete
+    gibbon.throws_exceptions = true
+  end
+
   def user_subscribe(name, email)
     merge_vars = {
       'FNAME' => name
     }
 
-    gibbon.lists.subscribe(
-      id: USER_LIST_ID,
-      email: { email: email },
-      merge_vars: merge_vars,
-      double_optin: true,
-      send_welcome: false
```

Comment:
From what I gathered from the docs they are settings on the list not Params you pass up in the subscribe call. I don't think I can see the prod lists so you'll have to poke around